addon/services/apollo.ts

```diff
@@ -71,23 +89,20 @@ export default Service.extend({
       return defaultOptions;
     }
     throw new Error('no Apollo service options defined');
-  }),
+  }
 
   init() {
-    this._super(...arguments);
+    super.init();
```

Comment:
Isn't it required to pass `...arguments` here?

Reply:
Indeed! 😊